docs/source/trainer/performance_tutorials/analyzing_traces.md

```diff
@@ -0,0 +1,56 @@
+# 🥽 Analyzing Traces
```

Comment:
Images in this doc look small on the web page, however, they are fine if opened in a new tab/or downloaded. Let us add a note mentioning this.

Reply:
I'll play with the `scale` attribute in the Sphinx `{image}` directive to see if helps here.  Otherwise we may need to take a new screenshot of these so the text is easier to read.

Reply:
Made it so the images can be clicked to enlarge, let me know what you think.

Reply:
Thanks. Much better now. Could you do the same for the one image in Profiling doc?